execution.go

```diff
@@ -69,9 +69,9 @@ func (s *ExecutableSchema) UpdateSchema(forceRebuild bool) error {
 	var services []*Service
 	var schemas []*ast.Schema
 	var updatedServices []string
-	var invalidschema float64 = 0
+	var invalidSchema float64 = 0
 
-	defer func() { promInvalidSchema.Set(invalidschema) }()
```

Comment:
Small nitpick: I'd make `invalidSchema` into a bool and coerce it to a `float` when calling `.Set()` on the Prometheus gauge.

Reply:
Ah yep will do, noticed it being odd it was a float when I first started but forgot to change it at the end.